source/background.js

```diff
@@ -123,7 +123,10 @@ function init() {
 	browser.runtime.onMessage.addListener(onMessage);
 	browser.runtime.onInstalled.addListener(handleInstalled);
 
-	browser.permissions.onAdded.addListener(addNotificationHandler);
+	// Chrome specific API
+	if (navigator.userAgent.includes('Chrome')) {
```

Comment:
This should be moved to a `util.js` file and be named `isChrome`, so we can use it elsewhere too. I think a better check would be `Boolean(window.chrome)`.

Reply:
If you wait a couple of weeks and comment on https://github.com/sindresorhus/refined-github/pull/2111, I’ll publish a polyfill for this soon. 😜

Reply:
> I think a better check would be `Boolean(window.chrome)`.

Unfortunately, `window.chrome` exists in Firefox too (for compatibility reasons), so  it had to be a UA sniff.